README.md

```diff
@@ -1,10 +1,10 @@
-# Basic Site Template
+# Standard Site Template
 
-This is the basic site template for Adobe Experience Manager as a Cloud Service (AEMaaCS).
+This is the standard site template for Adobe Experience Manager as a Cloud Service (AEMaaCS).
 
-<img src="previews/site.png?raw=true" alt="Basic site preview" width="50%">
+<img src="previews/site.png?raw=true" alt="Standard site preview" width="50%">
 
-It can serve as a most basic starting point for creating a new site, or as a staring point for creating custom site templates.
+It can serve as a standard starting point for creating a new site, or as a staring point for creating custom site templates.
```

Comment:
@comanV I know you didn't add it, but there's a typo here: "or as a starting point for creating"